kolibri/core/package.json

```diff
@@ -19,7 +20,6 @@
     "lodash": "^4.17.4",
     "loglevel": "^1.4.1",
     "markdown-it": "^8.3.1",
-    "rest": "^1.3.2",
```

Comment:
`js-cookie` dependency now also looks unused, and the `kolibri-core-for-export/package.json` needs to be updated as well.